src/urllib3/response.py

```diff
@@ -217,39 +237,39 @@ def get_redirect_location(self) -> typing.Union[str, "Literal[False]"]:
         return False
 
     @property
-    def data(self) -> bytes:
+    def data(self) -> Optional[Union[bytes, str]]:
```

Comment:
Should this be -> bytes?

Reply:
to be compatible with the `HTTPResponse.data` property.